### ISSUE TITLE
[BMG] test_torchinductor_opinfo native_batch_norm float16 comprehensive tests fail on BMG (pass on PVC) in torch-2.14

### ISSUE THREAD

**Issue by daisyden** [intel/torch-xpu-ops#5078]
## Summary

BMG-specific Inductor OpInfo failures observed with torch-2.14. These cases **pass on PVC** but fail on BMG, so they are hardware (BMG) specific.

## Affected cases

- `inductor/test_torchinductor_opinfo.py::TestInductorOpInfoXPU::test_comprehensive__native_batch_norm_legit_xpu_float16`
- `inductor/test_torchinductor_opinfo.py::TestInductorOpInfoXPU::test_comprehensive_native_batch_norm_xpu_float16`

## Environment

- Torch: 2.14
- Hardware: Intel BMG (fails)
- Reference: Intel PVC (passes)
- dtype: float16

## Reproduction

```bash
python test/inductor/test_torchinductor_opinfo.py TestInductorOpInfoXPU.test_comprehensive__native_batch_norm_legit_xpu_float16
python test/inductor/test_torchinductor_opinfo.py TestInductorOpInfoXPU.test_comprehensive_native_batch_norm_xpu_float16
```

## Notes

- BMG-only regression: both cases pass on PVC with the same torch-2.14 build.

**Comment by ZhaoqiongZ** [comment]
@torchxpubot fix

**Comment by torchxpubot** [comment]
## Result: `SKIPPED(no_longer_reproduces)` — no patch produced

**Branch:** none. No `fix_result*.json` was written, so the workflow's patch-export step has nothing to export — that is the correct outcome here, not a failure.

**Pipeline path:** triage (single-bug, agent-fixable) → reproduce (`NOT_REPRODUCED`) → stop. Stages 3–5 (root-cause / implement / verify) were never entered, per the skill's contract.

### What I found

The runner does have BMG hardware (Arc Pro B60, BMG-G31, 20 Xe cores / 160 EU), so the hardware precondition in the issue was satisfiable. Both reported tests passed:

- `2.15.0.dev20260906+xpu` (latest nightly, tree at pytorch main `a4cb62edee`) — 6 runs, all pass
- `2.14.0.dev20260811+xpu` (the reported 2.14 line, tree at that wheel's own commit `50e2fa0ee8`) — pass

Passing on the *reported* version matters: this is not "upstream fixed it since," it's "it does not reproduce on this BMG part at all." Neither op is in `inductor_one_sample["xpu"]`, so all OpInfo samples ran, not just the first.

I then instrumented the eager-vs-Inductor `assert_close` calls to measure tolerance headroom, because the established remedy for XPU fp16 failures in this test file is a `grad_atol`/`grad_rtol` bump in `inductor_override_kwargs["xpu"]` (pytorch/pytorch#191601, #194566, #195403). Across 104 comparisons per test the worst case consumed **13.8%** of its allowed tolerance (`max_abs 1.9e-6` vs `atol 1e-5 + rtol 1e-3`); everything else was under 0.6% or bit-exact. With ~7x headroom, a tolerance relaxation would have been a blind guess dressed up as a fix, so I did not write one.

Corroborating: no disabled-test issue exists upstream for either node id (pytorch's auto-disabler would have filed one), and neither test appears in `inductor_skips["xpu"]` or the xpu xfail dict — both are enabled and green upstream. CUDA, by contrast, skips all three batch-norm ops for `{f16, f32, f64}` as flaky.

### What I left undone, and why

I did not try reproducing under a whole-file run of `test_torchinductor_opinfo.py` (multi-hour) or under CI's parallel `-n`/worksteal layout. If the failure is an ordering/state effect, that gap is the likely place it hides — but the issue carries no traceback to justify spending hours on that guess.

The issue comment (edited in place, id `5563819909`) asks the reporter for the three things that would make a retry productive: the actual failure output (accuracy mismatch vs Triton/compile exception point to opposite fixes), the failing runner's exact BMG SKU and driver, and the full CI invocation.

---
_The [`fix` job](https://github.com/intel/torch-xpu-ops/actions/runs/34073729256) did not update this comment itself, so this is the agent's closing summary only._

**Comment by ZhaoqiongZ** [comment]
**strong-accepted** 

notes: no longer reproducible
